posts/2016-09-20-IDA2016-notes.md

```diff
@@ -0,0 +1,126 @@
+# International Data Week 2016 - Meeting Notes and Links
+
+*[International Data Week](internationaldataweek.org) in September 2016 brought together three events, [SciDataCon](http://www.scidatacon.org/), the International Data Forum, and the [Research Data Alliance 8th plenary](https://rd-alliance.org/plenaries/rda-eighth-plenary-meeting-denver-co). Joe Hand from the Dat team attended the conference and collected his notes here.*
+
+## SciDataCon2016
+
+SciDataCon featured sessions with 5-10 quick lightning talk each on a common theme. Notes from some of the more interesting sessions are below.
+
+### [Session on Sustainable Business Models for Data Repositories](http://www.scidatacon.org/2016/sessions/45/)
+
+The first two sessions covered the costs, budgets, and revenue streams for data repositories. Repositories have a wide range of pricing and revenue structures. The witness statements in the session link above provide details on the pricing models. Repositories are using a mix of subscriptions (paid by libraries, institutions, researchers), pay to download (to get specific datasets), or pay to deposit (pay to upload to the repository), and external funding (government, private).
+
+While each pricing model has its own strengths and downsides, the loser is clearly academic institutions, libraries, and researchers. Library managers and researchers expressed concern that they will be unable to keep up financially with the number of data repository subscriptions they need to access data. These costs are added onto the high costs institutions pay for journal subscriptions. Additionally, as more data becomes replicated in several places, it becomes unclear which repositories institutions should subscribe to.
+
+From the perspective of Dat, the costs were the most interesting part. Hosting and bandwidth for the data repositories amount to 1/4th to 1/3rd of their total costs. We hope that Dat will be able to better distribute or reduce a lot of those costs.
```

Comment:
why would Dat better distribute or reduce a lot of those costs? 

I would also remove the first sentence because the second sentence is a good enough segway from the last paragraph